vncdotool/api.py

```diff
@@ -117,6 +117,7 @@ def __dir__(self) -> List[str]:
 
 def connect(
     server: str,
+    username: Optional[str] = None,
```

Comment:
Adding a new parameter between of existing parameters breaks the API: Anyone doing a `connect("localhost:5900", "password") will now pass the 2nd argument to `username` instead.

Best append it at the end as the last parameter.

Reply:
Good catch, been a while since I've done python